### PR TITLE
Ability to filter # of results per page

### PR DESCRIPTION
**Pull request by FelixMalfait**

### CODE REVIEW

**Comment by ZenCocoon** [comment]
@FelixMalfait very nice to see your contribution :D Would you mind making clear that it can only be used to make it lower, not higher than the default please.

**Comment by FelixMalfait** [comment]
Sure 👍🏻

File: content/reference/index.md [hunk 1]
```diff
@@ -79,6 +79,8 @@ first | Shows the URL of the first page of results.
 prev  | Shows the URL of the immediate previous page of results.
 {: class="table table-bordered"}
 
+We've built in a default limit on results, you can use the `per_page` parameter to change how many results per page you want to get. Large queries can hurt performance, so `per_page` is capped by the default limit. If you wish to retrieve more records, you should make multiple  requests using the `page` parameter and combine the results within your application.
```
Comment (ZenCocoon) [inline content/reference/index.md]: minor typo: "double spacing" `multiple  requests`

Comment (FelixMalfait) [inline content/reference/index.md]: Haha nice spot you're though ;)

**Comment by ZenCocoon** [comment]
@FelixMalfait 🎉  it's now live, thanks a lot for helping us keeping the doc clean and complete.

**Pull request merged**